pkg/crc/machine/hyperkit/driver.go

```diff
@@ -31,8 +32,7 @@ func CreateHost(config config.MachineConfig) *hyperkitDriver {
 		},
 		Memory:      config.Memory,
 		CPU:         config.CPUs,
-		/* UUID will not be hardcoded in subsequent commits */
-		UUID:        "63655f17-7992-4f8e-8274-4aad9f7cf742",
+		UUID:        uuid.NewUUID().String(),
```

Comment:
so here we are generating random UUID which means it will also going to produce different mac address.

Reply:
We are generating a random UUID when `CreateHost` is called. This only happens on first `crc start` run, or after calling `crc delete`. If you only use `crc start`/`crc stop`, then we do not recreate the VM/regenerate a new UUID each run https://github.com/code-ready/crc/blob/master/pkg/crc/machine/machine.go#L88